ortho_processing/utils.py

```diff
@@ -87,7 +87,9 @@ def updateRecord(flight_dir, flight_id, status):
 def countFiles(path):
     file_count = 0
     for root, dirs, files in os.walk(path):
-        file_count += len(files)
+        for file in files:
```

Comment:
This won't match the file count we have in the database - since now you are counting all files (except hidden/desktop.ini) and not just images